static/app/components/actions/resolve.tsx

```diff
@@ -165,12 +193,12 @@ class ResolveActions extends React.Component<Props> {
 
         <MenuItemActionLink
           {...actionLinkProps}
-          title={t('Another version')}
+          title={t('Another existing release')}
```

Comment:
is `existing` needed? Seems a bit redundant. What is a non-existent release?

Reply:
A couple of customers and even some employees thought that they can enter there any version name (even future one) in which the issue will be resolved. (Especially in semver world)